src/Compilers/CSharp/CscCore/project.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "supports": {
-    "net46.app": { },
-    "dnxcore50.app": { }
+    "net46.app": {},
```

Comment:
The VS project.json editor prefers the space.  You might want to change whatever editor you are using so that this doesn't just go back and forth.